### ISSUE TITLE
GetAccountInfoAsync - unknown enum value in permissions field

### ISSUE THREAD

**Issue by TomasMichal** [JKorf/Binance.Net#958]
**Describe the bug**
An error "ArgumentNullException - Value cannot be null. (Parameter 'item')" occurs  when calling client.General.GetAccountInfoAsync().  The problem is that Binance returns the value "TRD_GRP_002" in the permissions field. I don't know what the value means, but it is there.

        "permissions": [
          "SPOT",
          "TRD_GRP_002"
        ]

[log.log](https://github.com/JKorf/Binance.Net/files/7785166/log.log)

**Comment by DanTheManGC** [comment]
I have started getting error when calling GetAccountInfoAsync also.  It started a few hours ago but only affects 1 of 2 accounts I am running.

I am using version 7.2.6 of Binance.NET and 13.0.1 of Newtonsoft.Json.

The error message is as follows (I have removed most balances for brevity)...



[4] Deserialize Unknown Exception: ArgumentNullException - Value cannot be null.
Parameter name: item
   at System.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.ThrowHelper.IfNullAndNullsAreIllegalThenThrow[T](Object value, ExceptionArgument argName)
   at System.Collections.Generic.List`1.System.Collections.IList.Add(Object item)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(IList list, JsonReader reader, JsonArrayContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.Linq.JToken.ToObject(Type objectType, JsonSerializer jsonSerializer)
   at Newtonsoft.Json.Linq.JToken.ToObject[T](JsonSerializer jsonSerializer)
   at CryptoExchange.Net.BaseClient.Deserialize[T](JToken obj, Nullable`1 checkObject, JsonSerializer serializer, Nullable`1 requestId)
, data: {
  "makerCommission": 10,
  "takerCommission": 10,
  "buyerCommission": 0,
  "sellerCommission": 0,
  "canTrade": true,
  "canWithdraw": true,
  "canDeposit": true,
  "updateTime": 1640741857882,
  "accountType": "SPOT",
  "balances": [
    {
      "asset": "BTC",
      "free": "0.00000000",
      "locked": "0.00000000"
    },
    {
      "asset": "LTC",
      "free": "0.00000000",
      "locked": "0.00000000"
    },
    {
      "asset": "RARE",
      "free": "0.00000000",
      "locked": "0.00000000"
    }
  ],
  "permissions": [
    "SPOT",
    "TRD_GRP_002"
  ]
}
